selfdrive/car/tests/test_fw_fingerprint.py

```diff
@@ -175,7 +175,7 @@ def test_fw_requests(self):
 
 class TestFwFingerprintTiming(unittest.TestCase):
   N: int = 5
-  TOL: float = 0.1
+  TOL: float = 0.12
```

Comment:
this test failed occasionally without this. also fails locally for me. Shane's looking to improve the tests: https://github.com/commaai/openpilot/pull/29712#issuecomment-1698732253